### PR TITLE
Initial implementation for global thought page

### PR DESCRIPTION
**Pull request by solocommand**
screencapture-0-0-0-0-12171-global-thought-leaders-2019-05-03-15_28_02

### CODE REVIEW

File: sites/waterworld/server/api/fragments/global-thought-leaders/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const company = require('./company');
```
Comment (zarathustra323) [inline sites/waterworld/server/api/fragments/global-thought-leaders/index.js]: You should be able to combine all these thought leader fragments into on with ... on calls

File: sites/waterworld/server/templates/website-section/global-thought-leaders.marko [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+<theme-pennwell-website-section-layout section=section>
+  <@head>
+    <cms-ad-gam-head />
+  </@head>
+  <@above-container>
+    <endeavor-gam-ad-unit-define-display name="lb1" modifiers=["top-of-page"] />
+  </@above-container>
+
+  <div class="page-wrapper default-page">
```
Comment (zarathustra323) [inline sites/waterworld/server/templates/website-section/global-thought-leaders.marko]: No need for default page here

File: sites/waterworld/server/templates/website-section/global-thought-leaders.marko [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+                    link-to=content.company
+                    modifiers=["center", "mh-100"]
+                    options=imageOpts
+                    use-placeholder=true
+                    position="top"
+                  />
+                  <@header-left>Watch ${content.company.name}'s Video</@header-left>
+                  <@description tag="div">
+                    <!-- @todo Bring in primary-video styling for item? -->
+                    <div class="content-page-body__primary-video mt-3">
```
Comment (zarathustra323) [inline sites/waterworld/server/templates/website-section/global-thought-leaders.marko]: No mt-3

File: sites/waterworld/server/templates/website-section/global-thought-leaders.marko [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+                <endeavor-item>
+                  <@image
+                    image=content.company.primaryImage
+                    link-to=content.company
+                    modifiers=["center", "mh-100"]
+                    options=imageOpts
+                    use-placeholder=true
+                    position="top"
+                  />
+                  <@header-left>Watch ${content.company.name}'s Video</@header-left>
```
Comment (zarathustra323) [inline sites/waterworld/server/templates/website-section/global-thought-leaders.marko]: No deep property access. Use the get helper

File: sites/waterworld/server/templates/website-section/global-thought-leaders.marko [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+            query-fragment=video
+          >
+            <div class="row">
+            <for|content| of=nodes>
+              <div class="col">
+                <endeavor-item>
+                  <@image
+                    image=content.company.primaryImage
+                    link-to=content.company
+                    modifiers=["center", "mh-100"]
```
Comment (zarathustra323) [inline sites/waterworld/server/templates/website-section/global-thought-leaders.marko]: Again, use a modifier that describes the element, not the properties it’s going to set. Also, no deep property access either.

File: sites/waterworld/server/templates/website-section/global-thought-leaders.marko [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+
+$ const { site } = out.global;
+$ const page = {
+  title: 'Global Thought Leader Series',
+  description: 'Certain companies in the water and wastewater industry are veritable trailblazers, forging ahead and sharing their expertise with others in an effort to help them stay current with new trends and technologies and do their jobs more efficiently. Our new Global Thought Leader Series will feature a collection of white papers and related videos from these innovators, giving you free and easy access to a bank of useful knowledge.',
+};
+$ const imageOpts = {
+  fit: 'clip',
+  h: 100,
+  minH: 100,
```
Comment (zarathustra323) [inline sites/waterworld/server/templates/website-section/global-thought-leaders.marko]: Are you sure you only want 100 pixels?

File: sites/waterworld/server/templates/website-section/global-thought-leaders.marko [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+            section-id=section.id
+            limit=2
+            include-content-types=["Whitepaper"]
+            query-fragment=whitepaper
+          >
+            <div class="row">
+            <for|content| of=nodes>
+              <div class="col">
+                <endeavor-item>
+                  <@header-left>Download ${content.company.name}'s Whitepaper</@header-left>
```
Comment (zarathustra323) [inline sites/waterworld/server/templates/website-section/global-thought-leaders.marko]: No deep property access

File: sites/waterworld/server/templates/website-section/global-thought-leaders.marko (hunk 2 above)
Comment (solocommand) [reply sites/waterworld/server/templates/website-section/global-thought-leaders.marko]: I did that for the 1.5rem margin-bottom that section pages don't have. Is there a better way to bring that in

File: sites/waterworld/server/templates/website-section/global-thought-leaders.marko (hunk 6 above)
Comment (solocommand) [reply sites/waterworld/server/templates/website-section/global-thought-leaders.marko]: Pretty sure...I want them to be the same height so they layout the same throughout the page.

**Comment by solocommand** [comment]
Needs base-cms/base-cms#76

**Pull request merged**